bot/exts/info/information.py

```diff
@@ -419,10 +419,14 @@ def format_fields(self, mapping: Mapping[str, Any], field_width: Optional[int] =
         return out.rstrip()
 
     @cooldown_with_role_bypass(2, 60 * 3, BucketType.member, bypass_roles=constants.STAFF_ROLES)
-    @group(invoke_without_command=True, enabled=False)
+    @group(invoke_without_command=True)
     @in_whitelist(channels=(constants.Channels.bot_commands,), roles=constants.STAFF_ROLES)
     async def raw(self, ctx: Context, *, message: Message, json: bool = False) -> None:
         """Shows information about the raw API response."""
+        if ctx.author not in message.channel.members:
+            await ctx.send(":x: You do not have permissions to see the channel this message is in.")
```

Comment:
Why not use an error embed here?

Reply:
Many other error messages in this cog use same format.

Reply:
That's a good point.